README.md

```diff
@@ -8,13 +8,17 @@ SPDX-License-Identifier: Apache-2.0
 -->
 
 # BenchExec
+
 ## A Framework for Reliable Benchmarking and Resource Measurement
 
 [![Build Status](https://gitlab.com/sosy-lab/software/benchexec/badges/main/pipeline.svg)](https://gitlab.com/sosy-lab/software/benchexec/pipelines)
 [![Apache 2.0 License](https://img.shields.io/badge/license-Apache--2-brightgreen.svg)](https://www.apache.org/licenses/LICENSE-2.0)
 [![PyPI version](https://img.shields.io/pypi/v/BenchExec.svg)](https://pypi.python.org/pypi/BenchExec)
 [![DOI](https://zenodo.org/badge/30758422.svg)](https://zenodo.org/badge/latestdoi/30758422)
 
+> [!NOTE]
```

Comment:
Interesting, I wasn't aware of this Markdown feature. Do you happen to know how supported this is outside of GitHub? We copy our Readme for example into the project description on PyPI, and I wonder whether this would look nice there or break rendering.

Reply:
I found it while searching for "nice" markdown boxes, I think this is a GitHub-specific feature, can remove it

Reply:
pandoc isn't aware of it, so I'll remove it